algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>I would put all the equations in the appendix (Technical details) and put here only the bottom line (with referring to the "Technical Details" section):</p><p><strong><em>The formula shows that the adiabatic error is minimized when:</em></strong></p><ol><li><br></li><li><br></li><li><br></li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>